### PR TITLE
feat(array-methods-2): animal object challenge

### PR DESCRIPTION
**Pull request by PReichetanz**
Challenge for JS Array Methods 2 Session.

rendered: https://github.com/neuefische/web-exercises/tree/feat/js-array-methods-2/animal-objects-challenge/sessions/js-array-methods-2/animal-objects

- [CSB](https://codesandbox.io/s/github/neuefische/web-exercises/tree/feat/js-array-methods-2/animal-objects-challenge/sessions/js-array-methods-2/animal-objects?file=/README.md)

refers to neuefische/web-curriculum-new-format#131

### CODE REVIEW

**Comment by CJAkkuec** [comment]
Hey there! I quite enjoyed this. :) Here's my [solution](https://codesandbox.io/s/hardcore-sky-f1crdv?file=/index.js)!

File: sessions/js-array-methods-2/animal-objects/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- `Array.some`()`
+- `Array.every`()`
+- `Array.reduce`()`
+- `Array.reverse`()`
+
+Additionally, at some point you will need:
+
+- `Array.filter`()`
+- `Array.map`()`
+- `String.startsWith`()`
```
Comment (djfarly) [inline sessions/js-array-methods-2/animal-objects/README.md:26]:
```suggestion
And now it's your turn: delete the value `null` and write code such that the variables have a value corresponding to their name. Use the following methods to achieve this goal:

- `Array.includes()`
- `Array.find()`
- `Array.findIndex()`
- `Array.sort()`
- `Array.some()`
- `Array.every()`
- `Array.reduce()`
- `Array.reverse()`

Additionally, at some point you will need:

- `Array.filter()`
- `Array.map()`
- `String.startsWith()`
```

File: sessions/js-array-methods-2/animal-objects/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Array Methods 2: An Array with Objects
+
+This codesandbox offers a some exercises to recap some enhanced array methods. Let's dive directly into it!
```
Comment (MerleMak) [inline sessions/js-array-methods-2/animal-objects/README.md:3]:
```suggestion
This CodeSandbox offers a some more exercises to recap some advanced array methods. Let's dive in!
```

**Review by MerleMak — APPROVED** [review]
Just some detail with the instruction phrase :)

**Pull request merged**